test/testfixture/customize_entity_callbacks.go

```diff
@@ -77,3 +77,12 @@ func TopologyDependency() CustomizeEntityCallback {
 func TopologyTree() CustomizeEntityCallback {
```

Comment:
is it used ?

Reply:
No it sits there to be used by anybody who wants to create a link type with tree topology and doesn't want to write the rather longish customize-entity-function. The whole idea was to have some ready-to-use customize-entity-functions.

Reply:
so I would not recommend to have it if it's not used for now